docs/manual/layout/site-structure/configure-pages.de.md

```diff
@@ -77,6 +73,124 @@ einer eindeutigen Beschreibung zu versehen.
+»firma.de« erreichbar ist, kannst du diese hier eingeben. Ruft ein Besucher dann »firma.de« in seinem Browser auf, wird 
+er automatisch zu dem entsprechenden Startpunkt einer Webseite weitergeleitet.
+
+**Protokoll (HTTPS verwenden):** Wenn deine Webseite über HTTPS verfügbar ist, muss diese Einstellung entsprechend
+konfiguriert werden. Ab Contao **4.10** heißt diese Einstellung **Protokoll** und lässt die Auswahl zwischen `http://`
+und `https://` zu. In Contao **4.9** werden Besucher automatisch auf HTTPS weitergeleitet, wenn diese Einstellung aktiv
+ist. Ab Contao **4.10** wird automatisch entweder zu `http://` oder `https://` weitergeleitet.
+
+{{< version "4.10" >}}
+
+**URL-Präfix:** Mit dieser Einstellung kann ein optionaler URL-Präfix allen Seitenaliasen unterhalb dieses Startpunkts
+zugewiesen werden. Diese Einstellung ist wie erwähnt ab Contao **4.10** verfügbar, davor gab es nur über die
+`contao.prepend_locale` Einstellung die Möglichkeit einen über die Sprache definierten Präfix zu verwenden. Nun ist dieser
+Präfix frei wählbar und damit unabhängig von der jeweils eingestellten Sprache.
+
+{{< version "4.10" >}}
+
+**URL-Suffix:** Mit dieser Einstellung kann der »URL-Suffix« geändert oder entfernt werden. Der URL-Suffix wird bei der
+Generierung der URL einer Seite an den Seitenalias angehängt.
+
+{{< version "4.5" >}}
+
+**Gültige Alias-Zeichen:** Der Slug-Generator ermöglicht es einen individuellen Zeichensatz für automatisch erstellte 
+Aliase auszuwählen.
+
+| Alias-Einstellungen                  | Erklärung                                                            |
+|:-------------------------------------|:---------------------------------------------------------------------|
+| Unicode-Zahlen und -Kleinbuchstaben  | Aus dem Seitennamen »Über uns« wird das Alias `über-uns` generiert.  |
+| Unicode-Zahlen und -Buchstaben       | Aus dem Seitennamen »Über uns« wird das Alias `Über-uns` generiert.  |
+| ASCII-Zahlen und -Kleinbuchstaben    | Aus dem Seitennamen »Über uns« wird das Alias `ueber-uns` generiert. |
+| ASCII-Zahlen und -Buchstaben         | Aus dem Seitennamen »Über uns« wird das Alias `Ueber-uns` generiert. |
+
+Für die Erzeugung des Aliases ist in Einzelfällen auch die eingestellte Sprache relevant. So wird ein Deutsches »Über« 
+zu »ueber« jedoch ein Finnisches »eläinkö« zu »elainko« konvertiert.
+
+{{% notice note %}}
+Diese Einstellung befindet sich in Contao **4.5** bis **4.9** in der Sektion _Alias-Einstellungen_.
+{{% /notice %}}
+
+{{< version "4.10" >}}
+
+**Ordner-URLs verwenden:** Wenn eine neue oder bestehende Seite ohne eingegebenen Alias abgespeichert wird, wird der Alias
+von Contao anhand des Seitennamens automatisch generiert. Ist diese Einstellung aktiv, wird bei der automatischen Generierung
+auch die Seitenhierarchie berücksichtigt. Dabei werden die Aliase der übergeordneten Seiten durch Slashes getrennt an den
```

Comment:
if I remember correctly, only the alias of the parent page will be used (which should already be a folder url with _its_ parent etc.)

Reply:
I know, but my thinking initially was that that's "too technical" for regular users 😁. But I think I'll update it anyway.

Reply:
I've now simply copy & pasted the same phrasing from the system settings. This way it is at least consistent.